src/nodes.coffee

```diff
@@ -3118,7 +3118,7 @@ UTILITIES =
 
   # Correctly set up a prototype chain for inheritance, including a reference
   # to the superclass for `super()` calls, and copies of any static properties.
-  extend: (o) -> "
+  extend: (o) -> """
```

Comment:
Oh, no, I was just figuring that multiline strings should always be block-quoted as a matter of style.